backend/src/main/java/gov/cdc/usds/simplereport/service/PatientBulkUploadService.java

```diff
@@ -114,48 +120,69 @@ public PatientBulkUploadResponse processPersonCSV(InputStream csvStream, UUID fa
-            org,
-            facility)) {
+            currentOrganization,
+            assignedFacility)) {
           continue;
         }
 
-        _personService.addPatient(
-            facilityId,
-            null, // lookupID
-            extractedData.getFirstName().getValue(),
-            extractedData.getMiddleName().getValue(),
-            extractedData.getLastName().getValue(),
-            extractedData.getSuffix().getValue(),
-            parseUserShortDate(extractedData.getDateOfBirth().getValue()),
-            address,
-            country,
-            List.of(
-                new PhoneNumber(
-                    parsePhoneType(extractedData.getPhoneNumberType().getValue()),
-                    extractedData.getPhoneNumber().getValue())),
-            parsePersonRole(extractedData.getRole().getValue(), false),
-            List.of(extractedData.getEmail().getValue()),
-            convertRaceToDatabaseValue(extractedData.getRace().getValue()),
-            convertEthnicityToDatabaseValue(extractedData.getEthnicity().getValue()),
-            null,
-            convertSexToDatabaseValue(extractedData.getBiologicalSex().getValue()),
-            parseYesNo(extractedData.getResidentCongregateSetting().getValue()),
-            parseYesNo(extractedData.getEmployedInHealthcare().getValue()),
-            null,
-            null);
+        // create new person with current organization, then add to new patients list
+        Person newPatient =
+            new Person(
+                currentOrganization,
+                assignedFacility,
+                null,
+                extractedData.getFirstName().getValue(),
+                extractedData.getMiddleName().getValue(),
+                extractedData.getLastName().getValue(),
+                extractedData.getSuffix().getValue(),
+                parseUserShortDate(extractedData.getDateOfBirth().getValue()),
+                address,
+                country,
+                parsePersonRole(extractedData.getRole().getValue(), false),
+                List.of(extractedData.getEmail().getValue()),
+                convertRaceToDatabaseValue(extractedData.getRace().getValue()),
+                convertEthnicityToDatabaseValue(extractedData.getEthnicity().getValue()),
+                null,
+                convertSexToDatabaseValue(extractedData.getBiologicalSex().getValue()),
+                parseYesNo(extractedData.getResidentCongregateSetting().getValue()),
+                parseYesNo(extractedData.getEmployedInHealthcare().getValue()),
+                null,
+                null);
+
+        // collect phone numbers and associate them with the patient, then add to phone numbers list
+        List<PhoneNumber> newPhoneNumbers =
+            _personService.assignPhoneNumbersToPatient(
+                newPatient,
+                List.of(
+                    new PhoneNumber(
+                        parsePhoneType(extractedData.getPhoneNumberType().getValue()),
```

Comment:
I think this `phoneNumberType` is optional in the CSV - just want to make sure we've tested what happens if an empty value is passed. I assume we just persist the phone type as `null` in the database, but it would be good to check.

Reply:
tl:dr though the function `assignPhoneNumbersToPatient` handles the null case

Reply:
@emmastephenson for this specific issue, I confirmed that our `parsePhoneType` function handles if phoneType is null. I added a [test](https://github.com/CDCgov/prime-simplereport/blob/d1ffe32080c182db48fa91bd5719b19d0aff0c8a/backend/src/test/java/gov/cdc/usds/simplereport/service/PatientBulkUploadServiceIntegrationTest.java#L102-L102) to make sure it handles this case, we were not testing this in any of the csv tests before.

Reply:
✅  unless we all feel strongly about refactoring this further.

Reply:
We don't allow multiple phone numbers in the csv! So I do think this might be more than we strictly need, but I also don't think we need to refactor further in this PR. We can always open up a separate ticket/PR for it later.